### DISCUSSION TITLE
Group inside group

### DISCUSSION THREAD

**Discussion by tomitomas** [Eugeny/tabby#9868] [General]
Hi there ! 

first of all : a big thank you for this amazing tools ! 
I've been using it for 1 week, and I love it ❤️ , I just sent a kofi as a small 💋 

Is there a way to set up group inside a group (on the profil page) ? 
if not, something you've already  though about ?

**Comment by Eugeny** [comment]
Unfortunately this is not possible yet - see #411

**Reply by tomitomas** [reply]
sorry, my quick search didn't pop out this thread. 

#411 was created 6 years ago, looks like the feature won't see the light of day right away :confused:
